src/Sentry/TransactionTracer.cs

```diff
@@ -395,4 +410,14 @@ public void Finish(Exception exception, SpanStatus status)
 
     /// <inheritdoc />
     public SentryTraceHeader GetTraceHeader() => new(TraceId, SpanId, IsSampled);
+
+    private void ReleaseSpans()
```

Comment:
Argh... this won't work actually. Once the tracer has finished it needs to be converted to an actual Transaction 😢